src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java

```diff
@@ -463,9 +462,9 @@ public void testOpenChannelSuccessResponse() throws Exception {
 
   @Test
   public void testInsertRow() {
-    SnowflakeStreamingIngestClientInternal<VectorSchemaRoot> client =
+    SnowflakeStreamingIngestClientInternal<ParquetChunkData> client =
```

Comment:
would this still work with Arrow?

Reply:
It wouldn't. So I changed this test. Other tests should not need any modification and work with both Arrow/Parquet as default versions.